### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by AlexForJava**

### CODE REVIEW

File: src/main/java/com/gmail/chernii/oleksii/YouTubeCommentThread.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    private String videoID;
+    private long maxSize;
+
+    public YouTubeCommentThread(String videoID, long maxSize) {
+        this.videoID = videoID;
+        this.maxSize = maxSize;
+    }
+
+    private void initialize() {
```
Comment (Taniann) [inline src/main/java/com/gmail/chernii/oleksii/YouTubeCommentThread.java]: firstly public methods then private
[ Ordering of class contents](https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents)

File: src/main/java/com/gmail/chernii/oleksii/Main.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package com.gmail.chernii.oleksii;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Created by Space on 04.04.2019.
+ */
+public class Main {
```
Comment (Taniann) [inline src/main/java/com/gmail/chernii/oleksii/Main.java]: better name with ending App

**Review by Taniann — CHANGES_REQUESTED** [review]
please make a little fixing

**Review by Taniann — APPROVED** [review]
good

**Comment by AlexForJava** [comment]
Thanks) 

8 апреля 2019, 11:03:04, от "Tania Nebesna" <notifications@github.com>:

@Taniann approved this pull request.

good

**Pull request merged**